### ISSUE TITLE
Evaluate model performance

### ISSUE THREAD

**Issue by jiuliangut** [jiuliangut/earthquake-monitor#149]
### Description

Add to the model training script to evaluate the performance of the model against the test set - using metrics such as RMS/MSE 

---

### Outcome

An updated model script which also carries out evaluation after the model has run, returning metrics which we can use to objectively measure how well the model predicts magnitude

---

### User Story

As a data engineer, I want to evaluate the performance of my ML model so that I know how well it actually predicts earthquake magnitude. This allows me to apply optimisation of hyper-parameters to fine-tune the model, increasing its precision.
